app/jobs/update_pharmgkb.rb

```diff
@@ -0,0 +1,50 @@
+require 'net/http'
+require 'openssl'
+require 'zip'
+
+OpenSSL::SSL::VERIFY_PEER = OpenSSL::SSL::VERIFY_NONE
+
+class UpdatePharmgkb < TsvUpdater
+  def tempfile_name
+    ['pharmgkb_interactions', '.tsv']
+  end
+
+  def importer
+    Genome::Importers::Pharmgkb::NewPharmgkb.new(tempfile)
+  end
+
+  def download_file
+    uri = URI('https://s3.pgkb.org/data/relationships.zip')
+    response = Net::HTTP.start(uri.host, uri.port,
+                    :use_ssl => uri.scheme == 'https') do |http|
+      request = Net::HTTP::Get.new uri
+      http.request(request)
+    end
+    Zip::InputStream.open(StringIO.new(response.body)) do |io|
+      while entry = io.get_next_entry
+        if entry.name == 'interactions.tsv'
+          tempfile.write(io.read.encode("ASCII-8BIT").force_encoding("utf-8"))
+        end
+      end
+    end
+  end
+
+  def next_update_time
```

Comment:
With the implementation of sidekiq, we no longer need this method. Instead you need to add an entry to `config/scheduled_tasks.yml`. The top list of entries in this file are the monthly tasks but with staggered start times.

Reply:
Removed `next_update_time` function and added task to update PharmGKB in `config/scheduled_tasks.yml`.